test/pkg/common/common.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import (
+	"context"
+	"testing"
+
+	"github.com/openshift-pipelines/pipelines-as-code/pkg/params"
+	"gotest.tools/v3/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func UpdateCM(ctx context.Context, t *testing.T, ns string, runcnx *params.Run, data map[string]string) {
```

Comment:
why don't you revert to the original configmap on defer ?

Reply:
since i need this for a e2e test i am writing, i'll be trying to add a new one

Reply:
Because its reverting when UpdateCM ends its functionality 

But i don't want that way

I want to revert the cm changes once after the main test scenario runs

Reply:
🤔 

it's what defer are for to end after the test

Reply:
see how i implemented here https://github.com/openshift-pipelines/pipelines-as-code/pull/1250/files#diff-ab0c34783b6ad30182f633bbedcd87b19c1dc6a6d163336b7e8982924502a121R15

and how i use it 

https://github.com/openshift-pipelines/pipelines-as-code/pull/1250/files#diff-91d7f3778d1b7a7b41ae3ab7d064590ddaade6f8ec8cc29033a23f4a5dc111e9R811

not the `defer` and `()` at the begin/end